ui/packages/consul-ui/app/components/consul/policy/list/index.hbs

```diff
@@ -8,13 +8,19 @@
   @items={{@items}}
 as |item|>
     <BlockSlot @name="header">
-{{#if (eq (policy/typeof item) 'policy-management')}}
+{{#if (or (eq (policy/typeof item) 'policy-management') (eq (policy/typeof item) 'read-only'))}}
         <dl class="policy-management">
           <dt>Type</dt>
           <dd>
-            <Tooltip>
-              Global Management Policy
-            </Tooltip>
+            {{#if (eq (policy/typeof item) 'policy-management')}}
+              <Tooltip>
+                Global Management Policy
+              </Tooltip>
+            {{else}}
+              <Tooltip>
+                Global Read-only Policy
+              </Tooltip>
+            {{/if}}
```

Comment:
Was going to suggest maybe we could simplify this a bit by doing most of this logic in the component

```js
get specialPolicyTooltip() {
  if (policyTypeOfItem('policy-management')) {
     return 'Global Management Policy';
  }

  if (policyTypeOfItem('read-only')) { 
    return 'Global Read-only policy';
  }
}
```

But I don't even see a component, I'm not sure how to call this helper outside of here, and this is the core-ui 😅, so feels not worth trying to change!

Just a thought though, definitely not blockable!